index.html

```diff
@@ -0,0 +1,262 @@
+                             <i class="fa fa-star" aria-hidden="true"></i>
+                           </div>
+                           <div class="card-review">
+                               <p class="card-dod-reviews">257 reviews</p>
+                           </div>
+                         </div>
+                         <div class="card-price">
+                             <p class="card-dod-discount-price">Rs. 3500/-</p>
+                             <p class="price-strike">Rs. 3999/-</p>
+                         </div>
+                         <div class="card-btn-container-hz">
+                           <button class="btn-icon-text"><span class="cart-icon">
+                             <i class="fa fa-shopping-cart" aria-hidden="true">
+                             </i></span>Add to Cart</button>
+                           <button class="btn-outline btn-medium">
+                             Add to wishlist</button>
+                         </div>
+                        </div>
+                    </div>
+
+                    <div class="card-text-container card-vertical">
+                        <div class="card-text-only">
+                         <div class="card-badge">Best Seller</div>
+                         <img class="card-img-vt"
+                          loading = "lazy"
+                          src="./assets/images/T-shirts/hamed-darzi-Psz62UPYx1E-unsplash.jpg"
+                          alt="image" />
+                         <h3 class="dod-company">Nike</h3>
+                         <h2 class="dod-product">Jacket</h2>
+                         <div class="card-rating">
+                           <div class="card-rating-stars">
+                             <i class="fa fa-star" aria-hidden="true"></i>
+                             <i class="fa fa-star" aria-hidden="true"></i>
+                             <i class="fa fa-star" aria-hidden="true"></i>
+                             <i class="fa fa-star" aria-hidden="true"></i>
+                             <i class="fa fa-star" aria-hidden="true"></i>
+                           </div>
+                           <div class="card-review">
+                               <p class="card-dod-reviews">257 reviews</p>
+                           </div>
+                         </div>
+                         <div class="card-price">
+                             <p class="card-dod-discount-price">Rs. 3500/-</p>
+                             <p class="price-strike">Rs. 3999/-</p>
+                         </div>
+                         <div class="card-btn-container-hz">
+                           <button class="btn-icon-text"><span class="cart-icon">
+                             <i class="fa fa-shopping-cart" aria-hidden="true">
+                             </i></span>Add to Cart</button>
+                           <button class="btn-outline btn-medium">
+                             Add to wishlist</button>
```

Comment:
`w` should be capital, looks good that way. You can use `text-transform` property for these buttons. So that you won't have to worry about uppercase and lowercase